### PR TITLE
Migrate to FloatingActionButtonSpeedDial

### LINKED ISSUE #1213: FABsMenu library has been deprecated
FABsMenu library has been deprecated, [FloatingActionSpeedDial](https://github.com/leinardi/FloatingActionButtonSpeedDial) is recommended for replacement, and issues  in FABsMenu might be enough of a problem to have to modify in release-3.3.0 branch.

### LINKED ISSUE #1800: Amaze aborts when clicking on menu for last item in app manager
**Describe the bug**

Clicking on last item in app manager will make the app abort
I have experience this bug a few times, checked in Google play and I saw no new upgrade available.
One can work around this bug by changing the order of the app list and click on the item.

**To Reproduce**

Open the app,
click on the top right 3-horizontal-bar icon to open its global menu,
scroll down and click open app manager,
scroll down to the last page,
click on the 3-vertical-dot icon on the right,
Amaze aborts.

**Expected behavior**
It should popup a menu for the user to choose from "Open, Backup, ..."

**Screenshots**
N/A

**Smartphone:**
 - Device: [XiaoMi Mi6X]
 - OS: [Android 9]
 - Rooted: [No]
 - Version: [3.3.4]

**Additional context**
No

### PR DESCRIPTION
**Pull request by TranceLove**
Fixes #1213.
Fixes #1800.

Also replaced `ic_clear_white` raster icon with `ic_close_white` vector icon.

### CODE REVIEW

**Comment by EmmanuelMess** [comment]
What's up with the buttons above the FAB?

 

They should be a bit smaller than the main FAB:

**Comment by TranceLove** [comment]
> What's up with the buttons above the FAB?
> 
> 

Let's see if the default behaviour helps. 1e47e74.

**Comment by VishalNehra** [comment]
Just tried, suggest to use current background with `MainActivity#showSmokeScreen` and `MainActivity#hideSmokeScreen`
Also, if FAB is open, shouldn't be able to press on list items, instead should close FAB on pressing outside.

File: app/src/main/java/com/amaze/filemanager/activities/MainActivity.java [hunk 1]
```diff
@@ -1494,58 +1498,38 @@ void initialiseFab() {
...
-        fabTitle.setRippleColor(Utils.getColor(this, R.color.white_translucent));
-        fabTitle.setOnClickListener(view -> {
-            mainActivityHelper.add(type);
-            floatingActionButton.collapse();
-        });
+        SpeedDialActionItem.Builder builder = new SpeedDialActionItem.Builder(id, icon)
+                .setLabel(fabTitle)
+                .setFabBackgroundColor(iconSkin);
 
         switch (getAppTheme().getSimpleTheme()) {
```
Comment (VishalNehra) [inline app/src/main/java/com/amaze/filemanager/activities/MainActivity.java]:
```
case LIGHT:
         fabBgView.setBackgroundResource(R.drawable.fab_shadow_light);
         break;
```

**Pull request merged**